index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>판다마켓</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <!-- Header -->
+  <header>
+    <a href="/15-Sprint-Mission/index.html"
+      ><img
+        src="images/logo/panda-market-logo.png"
+        alt="판다마켓 홈"
+        width="153"
+    /></a>  
+    <a href="login.html" id="loginButton" class="button">로그인</a>
+  </header>
+
+  <!-- Main -->
+  <main>
+    <!-- Home -->
+    <section id="home">
+      <div class="container">
+        <h1 id="homeDescription">
+          일상의 모든 물건을<br />
+          거래해 보세요
+        </h1>
+        <a href="items.html" class="button">구경하러 가기</a>
+      </div>
+    </section>
+
+    <!-- item -->
+    <section id="items" class="container">
+      <div class="item">
+        <img
+          src="images/home/feature1-image.png"
+          alt="인기 상품"
+          width="50%"
+        />
+        <div class="famousItem">
+          <h2 class="famousItemName">Hot item</h2>
+          <h1>인기 상품을<br />확인해 보세요</h1>
+          <p class="sectionDescription">
+            가장 HOT한 중고거래 물품을<br />판다마켓에서 확인해 보세요
+          </p>
+        </div>
+      </div>
+      <div class="item">
+        <div class="famousItem">
+          <h2 class="famousItemName">Search</h2>
+          <h1>구매를 원하는<br />상품을 검색하세요</h1>
+          <p class="sectionDescription">
+            구매하고 싶은 물품은 검색해서
+            <br />쉽게 찾아보세요
+          </p>
+        </div>
+        <img
+          src="images/home/feature2-image.png"
+          alt="검색"
+          width="50%"
+        />
+      </div>
+      <div class="item">
+        <img
+          src="images/home/feature3-image.png"
+          alt="판매 상품"
+          width="50%"
+        />
+        <div class="famousItem">
+          <h2 class="famousItemName">Register</h2>
+          <h1>판매를 원하는<br />상품을 등록하세요</h1>
+          <p class="sectionDescription">
+            어떤 물건이든 판매하고 싶은 상품을
+            <br />쉽게 등록하세요
+          </p>
+        </div>
+      </div>
+    </section>
+    <section id="bottom">
+      <div class="container">
+        <h1>
+          믿을 수 있는
+          <br />
+          판다마켓 중고거래
+        </h1>
+      </div>
+    </section>
+  </main>
+
+  <!-- Footer -->
+  <footer>
+    <div>©codeit - 2024</div>
+    <div id="footerMenu">
+      <a href="privacy.html">Privacy Policy</a>
+      <a href="faq.html">FAQ</a>
+    </div>
+    <div id="sns">
+      <a
+        href="https://www.facebook.com/"
+        target="_blank"
+        rel="noopener noreferrer"
+        ><img src="images/social/facebook-logo.svg" alt="페이스북" width="20"
+      /></a>
+      <a href="https://twitter.com/" target="_blank" rel="noopener noreferrer"
+        ><img src="images/social/twitter-logo.svg" alt="트위터" width="20"
+      /></a>
```

Comment:
💊 제안
현재 코드에서는 하나의 태그가 나눠져 있어 가독성이 조금 떨어질 수 있습니다.  
태그 간 계층 구조를 명확하게 표현하기 위해 아래처럼 작성하시는 것을 추천드려요~

```suggestion
      <a href="https://twitter.com/" target="_blank" rel="noopener noreferrer">
        <img src="images/social/twitter-logo.svg" alt="트위터" width="20" />
      </a>
```